index.html

```diff
@@ -83,13 +83,36 @@ <h3>Сувениры</h3>
                 </ul>
             </section>
             
-            <section class="search">
-                <h2>Поиск гостиницы в Седоне</h2>
+            <section class="search-title">
+                <h2 class="visually-hidden">Поиск гостиницы в Седоне</h2>
                 <b>Заинтересовались?</b>
                 <p>Укажите предполагаемые даты поездки, и мы покажем вам лучшие предложения гостиниц в Седоне</p>
                 <a class="button" href="catalog.html">Поиск гостиницы в Седоне</a>
                 <img src="img/map.jpg" width="1200" height="594" alt="Где находится Седона">
-            </section>       
+            </section>  
+            
+            <section class="modal search-modal">
+                <h2 class="visually-hidden">Форма поиска гостиниц в Седоне</h2>
+                <form class="search-form" action="https://echo.htmlacademy.ru" method="get">
+                    <p class="search-criteria">
+                        <label for="arrival-date">Дата заезда:</label>
+                        <input class="visually-hidden criteria-input" type="text" name="arrival-date" id="arrival-date" value="" placeholder="24 апреля 2017">
+                    </p>
+                    <p class="search-criteria">
+                        <label for="departure-date">Дата выезда:</label>
+                        <input class="visually-hidden criteria-input" type="text" name="departure-date" id="departure-date" value="" placeholder="4 июля 2017">
+                    </p>
+                    <p class="search-criteria">
+                        <label for="adults">Взрослые:</label>
+                        <input class="visually-hidden criteria-input" type="number" name="adults" id="adults" value="" step="1" min="0" placeholder="2">
```

Comment:
Так как средствами css стилизовать переключатели у input type number нельзя, нужно заменить, на type text и добавить две кнопки + и -
```suggestion
                        <button type="button">-</button><input class="visually-hidden criteria-input" type="text" name="adults" id="adults" value="" step="1" min="0" placeholder="2"><button type="button">+</button>
```